<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="1" />
  <arg name="timeout" default="0.5" />
  <arg name="chain_start" default="r_foot" />
  <arg name="chain_end" default="pelvis" />
  <arg name="urdf_param" default="$(find robot_gazebo)/urdf/legs11.URDF" />

  <param name="robot_description" textfile="$(find robot_gazebo)/urdf/legs11.URDF" />


  <node name="control_akdl_demo" pkg="control_demo" type="control_akdl_demo" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
  	<param name="urdf_param" textfile="$(arg urdf_param)" />
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
	<!--    <param name="urdf_param" value="$(param robot_description)"/> -->
  </node>

</launch>
